src/node.rs

```diff
@@ -467,6 +467,15 @@ impl Node {
             elder.trigger_relocation(node_id, destination_address)
         }
     }
+
+    /// Check whether the peer is an elder in our section.
+    pub fn is_peer_our_elder(&self, pub_id: &PublicId) -> bool {
+        if let Some(elder) = self.elder_state() {
+            elder.is_peer_our_elder(pub_id)
+        } else {
+            false
+        }
```

Comment:
Looks fine, I kinda like the chained formulation but that's of course highly arbitrary ;)
```rust
self.elder_state().map(|elder| elder.is_peer_our_elder(pub_id)).unwrap_or_false()
```